wiki/Music_theory/Beat/id.md

```diff
@@ -0,0 +1,5 @@
+# Ketukan (Beat)
+
+Sebuah **ketukan (beat)** adalah satuan waktu yang berulang dalam musik, khususnya denyut nadi musik yang diketuk oleh pendengar. Di osu!, kamu dapat membandingkannya dengan tanda tick putih di [editor timeline](/wiki/Beatmap_Editor/Timelines) yang menjadi kekuatan irama lagu.
```

Comment:
this translation feels very different w/ the other one you have tho, might as well making everything more consistent between the two

Reply:
yeah it should.